lib/registrant.rb

```diff
@@ -0,0 +1,30 @@
+class Registrant 
+    attr_reader :age,
+                :name
+
+    attr_accessor :license_data
+
+    def initialize(name, age, permit = false)
+        @name = name
+        @age = age
+        @permit = permit
+        @license_data = {written: false, 
+                        license: false, 
+                        renewed: false}
+    end
+
+    def permit?
+        @permit = @permit
```

Comment:
This method is a boolean method, so we just need it to return true or false depending on the value of @permit. @permit is a boolean attribute, so that means that this `permit?` method can just contain `@permit` since that will evaluate to either true or false! We don't need to assign it to anything in this method